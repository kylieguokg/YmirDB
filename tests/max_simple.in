max no
set a
get a
max a
push a 28
get a
max a
set entry -1999 2993 200 123456789 0
get entry
max entry
set entry -10 -10 -10
get entry
max entry
bye
