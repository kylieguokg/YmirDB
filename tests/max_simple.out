> no such key

> ok

> []

> empty entry

> ok

> [28]

> 28

> ok

> [-1999 2993 200 123456789 0]

> 123456789

> ok

> [-10 -10 -10]

> -10

> bye
